SOC/common/simulation.py

```diff
@@ -5,38 +5,45 @@
 import matplotlib.pyplot as plt
 from matplotlib import animation
 import pandas
-import seaborn
 import zarr
 import datetime
+import typing
 
 class Simulation:
-    """Base class for SOC simulations."""
+    """Base class for SOC simulations.
+
+    :param L: linear size of lattice, without boundary layers
+    :type L: int
+    :param save_every: number of iterations per snapshot save
+    :type save_every: int or None
+    :param wait_for_n_iters: How many iterations to skip to skip before saving data?
+    :type wait_for_n_iters: int
+    """
     values = NotImplemented
     saved_snapshots = NotImplemented
 
     BOUNDARY_SIZE = BC = 1
     def __init__(self, L: int, save_every: int = 1, wait_for_n_iters: int = 10):
-        """__init__
-
-        :param L: linear size of lattice, without boundary layers
-        :type L: int
-        :param save_every: number of iterations per snapshot save
-        :type save_every: int or None
-        """
         self.L = L
         self.visited = np.zeros((self.L_with_boundary, self.L_with_boundary), dtype=bool)
         self.data_acquisition = []
         self.save_every = save_every
         self.wait_for_n_iters = wait_for_n_iters
         # zliczanie relaksacji
-        self.releases = np.zeros((self.L_with_boundary, self.L_with_boundary), dtype=int)
+        self.releases = np.zeros((self.L_with_boundary, self.L_with_boundary), dtype=int) # TODO przenieść konkretnie do OFC?
```

Comment:
Czyli uznać to jako wielkość dot. tylko tego modelu? W sumie nie stosowaliśmy poza tym.

Reply:
to dodałem do BTW, chociaż jaka korzyść z tego niewiadoma.

Reply:
No właśnie nie wiem, jeśli jest w tamtym PDFie  że w BTW też się to liczy, to można zostawić - chociaż trochę mnie boli że ta macierz jest inicjalizowana też dla ForestFire i Manny, które jej się nie przydają. Oczywiście puszczamy za małe symulacje, żeby to było istotne, ale jednak - w imię zasad. ;)